docs/authoring.md

```diff
@@ -101,7 +101,9 @@ const {{ channelName | camelCase }} = '{{ channelName }}';
 
 ## Hooks
 
-Hooks are functions called by the generator on a specific moment in the generation process. For now there is one hook supported called `generate:after` that is called at the very end of the generation. The generator will parse all the files in the `.hooks` directory.
+Hooks are functions called by the generator on a specific moment in the generation process. 
+For now there is two hooks supported called `generate:before` and `generate:after` that are called before processing the template and at the very end of the generation respectively. 
```

Comment:
I think we can convert this into a table for better readability. Also, I'm sure new hooks will be added in the future. It will make things easier :)

Reply:
@fmvilas please check new version